### ISSUE TITLE
URL to enable / disable Motion notifications ?

### ISSUE THREAD

**Issue by Oliv59** [ccrisan/motioneye#1223]
Hello there,

Slightly different from the other topics i found, i don't want to enable or disable the motion detection, but just the detection notifications (email, and run a command).
Of course, without losing all inside setup of those functionalities, to be able to reactivate them still by a simple URL calling, if that's possible ?
Thanks,
Olivier.

**Comment by corvy** [comment]
I wonder the same. I want to be able to enable "away" mode from Home Assistant for instance when there is no more movement in the living-room or the alarm is turned on then I want notifications on the camera to be enabled.

**Comment by aredon** [comment]
This is possible through motion itself. You need to turn off the localhost limitation so you can see it from another computer. Then you can navigate to things like camera status or enable/disable motion detection. I've used it in my home detection script which is working extremely well.

https://motion-project.github.io/motion_config.html

Read up on the web control section. Here's an example url for camera 0

http://localhost:8080/0/detection/pause

**Comment by aredon** [comment]
You'll have to write your own script for this. You could do a simple apache web page to toggle a file or a database entry to determine on or off.

Then have your end command check before doing anything. Motioneye doesn't provide this functionality. 

I have a home detection script I've been using on mine to turn motion detection on and off (and thus also notifications).

**Comment by corvy** [comment]
Yes I ended up using the same way, disable motion on motioneye but keep the camera doing motion detection over MQTT so I could continue to use the camera as a motion detection sensor even when turned off on motioneye.
